scripts/cake.lic

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# Script for automatically summoning a default manna bread, or a random one depending on preference.
+#
+# Author: Ineum
+# Game: Gemstone IV
+# tags: magic, manna
+# Version: 1.0
+
+Vars[@script_name] ||= { manna_type: 'potato cake' }
+
+summon_type = Vars[@script_name][:manna_type]
```

Comment:
I don't think you can use `@` instance variables in an ambient context like this, won't `@script_name` always be `nil`?  Fairly sure this should be `Script.current.name`

Reply:
@sandersch thoughts?  You're the Ruby guru :slightly_smiling_face:

Reply:
Good catch - fixed!